WebAPI/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using App.DAL.Entities;
+using AutoMapper;
+using CommonDbProperties.Interfaces.Repositories;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using WebAPI.Dtos;
+
+namespace WebAPI.Controllers;
+
+[ApiController]
+[ApiVersion("1.0")]
+[Route("api/v{version:apiVersion}/[controller]")]
+public class CategoryController
```

Comment:
Každý controller by mal dediť z ControllerBase

```suggestion
public class CategoryController : ControllerBase
```